### PR TITLE
Pushnotes

### PR DESCRIPTION
**Pull request by jeroenleenarts**

### CODE REVIEW

File: CocoaHeadsNL/CocoaHeadsNL/AppDelegate.swift [hunk 1]
```diff
@@ -40,8 +58,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         CKContainer.default().add(badgeResetOperation)
     }
 
+    // MARK: Notifications
+    
     func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
-
+
```
Comment (jeroenleenarts) [inline CocoaHeadsNL/CocoaHeadsNL/AppDelegate.swift]: @barthoffman Here.

Comment (barthoffman) [reply CocoaHeadsNL/CocoaHeadsNL/AppDelegate.swift]: Note that CloudKit does handle device tokens for you, so you don't need to implement the application(_:didRegisterForRemoteNotificationsWithDeviceToken:) method, unless you need that for another purpose.

**Review by barthoffman — COMMENTED** [review]
Already found a way to remove subscriptions in case user disables push notifications. So pull request can be deleted to do a different one :)

File: CocoaHeadsNL/CocoaHeadsNL.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -394,6 +394,7 @@
 						CreatedOnToolsVersion = 6.1.1;
 						DevelopmentTeam = D55PL7PU7S;
 						LastSwiftMigration = 0900;
+						ProvisioningStyle = Automatic;
```
Comment (jeroenleenarts) [inline CocoaHeadsNL/CocoaHeadsNL.xcodeproj/project.pbxproj:397]: We gebruiken Fastlane.

File: CocoaHeadsNL/CocoaHeadsNL.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -853,13 +854,15 @@
 				CODE_SIGN_ENTITLEMENTS = CocoaHeadsNL/CocoaHeadsNL.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
-				DEVELOPMENT_TEAM = D55PL7PU7S;
+				CODE_SIGN_STYLE = Automatic;
```
Comment (jeroenleenarts) [inline CocoaHeadsNL/CocoaHeadsNL.xcodeproj/project.pbxproj:857]: We are using Fastlane for provisioning.

File: CocoaHeadsNL/CocoaHeadsNL.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -876,15 +879,17 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = CocoaHeadsNL/CocoaHeadsNL.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
-				DEVELOPMENT_TEAM = D55PL7PU7S;
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CODE_SIGN_STYLE = Automatic;
```
Comment (jeroenleenarts) [inline CocoaHeadsNL/CocoaHeadsNL.xcodeproj/project.pbxproj:883]: We are using fastlane for provisioning.

File: CocoaHeadsNL/CocoaHeadsNL/AppDelegate.swift [hunk 5]
```diff
@@ -40,9 +71,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         CKContainer.default().add(badgeResetOperation)
     }
 
-    func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
-
-    }
+    // MARK: Notifications
+    
+//    func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
```
Comment (jeroenleenarts) [inline CocoaHeadsNL/CocoaHeadsNL/AppDelegate.swift]: This is not needed anymore since you are using iOS 10 style notifications. Let's remove this then.

**Review by jeroenleenarts — COMMENTED** [review]
Please check that provisioning with fastlane still works.

File: CocoaHeadsNL/CocoaHeadsNL/AppDelegate.swift (hunk 5 above)
Comment (barthoffman) [reply CocoaHeadsNL/CocoaHeadsNL/AppDelegate.swift]: It is needed if you want to "receive" push notifications from other sources than cloud kit ;)

**Pull request closed without merging**